src/components/exchange/ExchangeInput.js

```diff
@@ -84,27 +96,46 @@ const ExchangeInput = (
     },
     [onFocus]
   );
+  let valueToRender = value;
+  if (value?.length > androidMaskMaxLength) {
+    valueToRender = value.substring(0, androidMaskMaxLength) + '...';
+  }
 
   return (
-    <Input
-      {...props}
-      color={color}
-      editable={editable}
-      keyboardAppearance={keyboardAppearance}
-      letterSpacing={letterSpacing}
-      mask={mask}
-      onBlur={handleBlur}
-      onChange={handleChange}
-      onChangeText={handleChangeText}
-      onFocus={handleFocus}
-      placeholder={placeholder}
-      placeholderTextColor={placeholderTextColor}
-      ref={ref}
-      size={size}
-      testID={testID}
-      value={value}
-      weight={weight}
-    />
+    <React.Fragment>
+      <Input
+        {...props}
+        color={color}
+        editable={editable}
+        keyboardAppearance={keyboardAppearance}
+        letterSpacing={letterSpacing}
+        mask={mask}
+        onBlur={handleBlur}
+        onChange={handleChange}
+        onChangeText={handleChangeText}
+        onFocus={handleFocus}
+        placeholder={placeholder}
+        placeholderTextColor={placeholderTextColor}
+        ref={ref}
+        size={size}
+        testID={testID}
+        value={value}
+        weight={weight}
+      />
+      {useCustomAndroidMask && !ref.current?.isFocused() && (
+        <AndroidMaskWrapper>
+          <Text
+            letterSpacing={letterSpacing}
+            size={size}
+            testID={testID}
+            weight={weight}
+            {...props}
+          >
+            {valueToRender}
+          </Text>
+        </AndroidMaskWrapper>
+      )}
+    </React.Fragment>
```

Comment:
I think we're using more `<>`

Reply:
41 files use `<React.Fragment>` and only 12 use `<>`